CMakeLists.txt

```diff
@@ -830,6 +830,21 @@ if(ENABLE_ROCM)
   option(QMC_DISABLE_HIP_HOST_REGISTER "Disable hipHostRegister for pinning host memory" ON)
 endif(ENABLE_ROCM)
 
+if(USE_NVTX_API AND QMC_CUDA2HIP)
+  message(STATUS "Enabling use of ROCm rocTX APIs")
+  find_library(
+    ROCTX_API_LIB NAME libroctx64.so
+    HINTS ${ROCM_PATH}
```

Comment:
We don't respect `ROCM_PATH`, please remove.

Reply:
Sorry for that.
Removed.